### PR TITLE
Added history page

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
History page added : 

Videos viewed by the user will be automatically added to the history page 
The user can remove the videos from their history 

Screenshot (268)

### CODE REVIEW

**Review by hrishi82 — APPROVED** [review]

File: TravelBug/src/Components/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -10,7 +10,9 @@ const Sidebar = () => {
           <div className="component-child">
             <i class="fas fa-home"></i>
```
Comment (mdarif6) [inline TravelBug/src/Components/Sidebar/Sidebar.jsx:11]: - you can change class of <i></> too in className. other wise it will give error in console

**Review by mdarif6 — APPROVED** [review]
- rest looking good.
-  ready to merge

**Review by anuj-kumary — APPROVED** [review]
- Functionality working as excepted.
- Clean code and good use of naming convention
Good to merge

File: TravelBug/src/Components/Sidebar/Sidebar.jsx (hunk 1 above)
Comment (Shah-Arsalan) [reply TravelBug/src/Components/Sidebar/Sidebar.jsx:11]: Done

**Pull request merged**